src/core/stream.h

```diff
@@ -860,6 +860,16 @@ QuicStreamIndicateSendShutdownComplete(
     _In_ BOOLEAN GracefulShutdown
     );
 
+//
+// Enqueues a SendRequest from the temporary queue to the actual queue.
+//
+_IRQL_requires_max_(PASSIVE_LEVEL)
+void
+QuicStreamEnqueueSendRequest(
+    _In_ QUIC_STREAM* Stream,
+    _Inout_ QUIC_SEND_REQUEST* SendRequest
+);
```

Comment:
```suggestion
_IRQL_requires_max_(PASSIVE_LEVEL)
void
QuicStreamEnqueueSendRequest(
    _In_ QUIC_STREAM* Stream,
    _Inout_ QUIC_SEND_REQUEST* SendRequest
    );
```

Reply:
For functions only called in one file, the declaration usually is just put at the top of that file, instead of the header. The header is so other files can call it.

Reply:
Moved the declaration! 2d8415f6bf02af962e461fc41b8ef9b23b9ab669